.env.sample

```diff
@@ -19,3 +19,5 @@ ENABLE_FUNCTION_GRAPHQL="true"
 MONGODB_URL="mongodb://127.0.0.1:27050/astroport"
 MONGODB_URL_R="mongodb://127.0.0.1:27050/astroport"
 MONGODB_URL_RW="mongodb://127.0.0.1:27050/astroport"
+
+TERRA_HIVE_ENDPOINT="http://localhost:8085/graphql"
```

Comment:
For the sake of consistency, should this be expressed as `127.0.0.1` like above?